tools/run-converter.sh

```diff
@@ -32,5 +32,10 @@ then
       echo "status=failed" >> $GITHUB_OUTPUT 
   fi
 else
+  if [ "$?" -eq "0" ]
```

Comment:
Not sure whether we actually need to do anything here - I think the console output is generally fairly obvious. (Not sure if that's true of the section renumbering tool.)

Reply:
Likely true. I was using it to test and make sure the "else" branch was hit. I'll clean those up.

I think all are unnecessary.